draft-ietf-ppm-dap.md

```diff
@@ -120,6 +120,13 @@ input is ever seen in the clear by any aggregator.
 
 (\*) Indicates a change that breaks wire compatibility with the previous draft.
 
+12:
+
+- Rename the fixed-size query type to leader-selected, to align the name with
+  the behavior of this query type.
+
+- Remove the `max_batch_size` parameter of the leader-selected query type.
```

Comment:
nit: I'd re-order these, since you need the second as context for the first.

Reply:
Done, though now we're referencing "fixed-size" directly before we mention that it is renamed.